src/ResourceManager/Common/Commands.ResourceManager.Common/AzureRMCmdlet.cs

```diff
@@ -296,5 +296,21 @@ protected override void BeginProcessing()
 
             base.BeginProcessing();
         }
+
+        protected void ThrowInvalidArgumentError(string errorMessage, string arg)
+        {
+            ThrowTerminatingError
+                (new ErrorRecord(
+                    new ArgumentException(string.Format(CultureInfo.InvariantCulture,
+                        errorMessage, arg)),
+                    "InvalidArgument",
+                    ErrorCategory.InvalidArgument,
+                    null));
+        }
+
+        protected string GetDiskNameFromId(string Id)
```

Comment:
Is this utility function going to be user by cmdlets outside compute as well? If n ot, then I think a better place to put it would be under [compute common](https://github.com/Azure/azure-powershell/tree/preview/src/ResourceManager/Compute/Commands.Compute/Common), more specifically the [compute base cmdlet](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute/Common/ComputeClientBaseCmdlet.cs)